### PR TITLE
Check for queued for good measure and use greater of eta or now

### PR DESCRIPTION
**Pull request by jflatow**

### CODE REVIEW

File: scenario/utils/index.ts [hunk 1]
```diff
@@ -350,18 +350,19 @@ export async function executeOpenProposal(
...
-  const proposal = await governor.proposals(id);
-  await setNextBlockTimestamp(dm, proposal.eta.toNumber() + 1);
-
-  // Execute proposal (w/ gas limit so we see if exec reverts, not a gas estimation error)
-  await setNextBaseFeeToZero(dm);
-  await governor.execute(id, { gasPrice: 0, gasLimit: 12000000 });
+  // Execute proposal (maybe, w/ gas limit so we see if exec reverts, not a gas estimation error)
+  if (await governor.state(id) == ProposalState.Queued) {
+    const block = await dm.hre.ethers.provider.getBlock();
+    const proposal = await governor.proposals(id);
```
Comment (kevincheng96) [inline scenario/utils/index.ts:361]: Not sure why this is complaining and failing unit tests.

File: scenario/utils/index.ts [hunk 2]
```diff
@@ -350,18 +350,19 @@ export async function executeOpenProposal(
...
 
-  const proposal = await governor.proposals(id);
-  await setNextBlockTimestamp(dm, proposal.eta.toNumber() + 1);
-
-  // Execute proposal (w/ gas limit so we see if exec reverts, not a gas estimation error)
-  await setNextBaseFeeToZero(dm);
-  await governor.execute(id, { gasPrice: 0, gasLimit: 12000000 });
+  // Execute proposal (maybe, w/ gas limit so we see if exec reverts, not a gas estimation error)
+  if (await governor.state(id) == ProposalState.Queued) {
+    const block = await dm.hre.ethers.provider.getBlock();
```
Comment (kevincheng96) [inline scenario/utils/index.ts:360]: Oh I think `getBlock()` takes a `blockNumber` as an argument.

File: scenario/utils/index.ts (hunk 1 above)
Comment (kevincheng96) [reply scenario/utils/index.ts:361]: Actually see https://github.com/compound-finance/comet/pull/723/files#r1145304993.

**Review by kevincheng96 — APPROVED** [review]
👍

**Review by scott-silver — APPROVED** [review]

**Pull request merged**